packages/dynamic-adapter/src/index.ts

```diff
@@ -127,7 +127,10 @@ class DynamicAdapter implements IDynamicAdapter {
                 return;
             }
 
-            const node = (insPoint.selector) ? contextNode.querySelector(insPoint.selector) as HTMLElement : contextNode;
+            const node = (insPoint.selector) ? contextNode.querySelector(insPoint.selector) as HTMLElement
```

Comment:
*Dima's proposal:*
When does the insertion point not have a selector? ((Seen only in shared adapter))
Could we potentially use this case as a check for context-oriented instance points?